account_invoice_ubl_email_attachment/models/mail_template.py

```diff
@@ -26,7 +26,9 @@ def generate_email(self, res_ids, fields=None):
                     ("res_model", "=", "account.move"),
                     ("res_id", "=", res_id),
                     ("name", "=", ubl_filename),
-                ]
+                ],
+                order="create_date desc",
+                limit=1,
```

Comment:
If you are adding the limit, then the `if len(ubl_attachments) == 1` has no sense.

Can you explain why the limit=1 is important?

Reply:
See https://github.com/OCA/edi/pull/287, if we dont have the limit then all the previous versions of the UBL will also be attached, which makes no sense. I agree the code can be shortened though, will push a fixup.